v2/fb-xcom-api-validator/src/server/middlewares/timestamp-middleware.ts

```diff
@@ -0,0 +1,41 @@
+import logger from '../../logging';
+import { IncomingHttpHeaders } from 'http';
+import { FastifyReply, FastifyRequest, HookHandlerDoneFunction } from 'fastify';
+import config from '../../config';
+import { BadRequestError, RequestPart } from '../../client/generated';
+
+const log = logger('middleware:timestamp');
+
+const EXPIRED_REQUEST_ERROR: BadRequestError = {
+  message: 'Expired request',
```

Comment:
"Expired" is too ambiguous IMHO. Let's be more descriptive; something like "Request timestamp header is too old".